{"uuid": "c33c85ea-c0d3-4e7d-9834-a3276048d9f7", "children": ["f67c478a-f9d3-4b79-9617-9e8b835659a0", "0672f595-092d-4ce6-8cd5-6fa12c4bca22"], "befores": [{"name": "log_on_failure", "status": "passed", "start": 1737288320219, "stop": 1737288320220}], "afters": [{"name": "log_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Class' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\Vishal\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Vishal\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 922, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\Vishal\\PycharmProjects\\pytestautomationdemo\\Pages\\LoginPage.py\", line 30, in log_on_failure\n    if item.rep_call.failed:\n       ^^^^^^^^^^^^^\n"}, "start": 1737288347673, "stop": 1737288347680}], "start": 1737288320219, "stop": 1737288347680}